lib/g_searcher/search_results.ex

```diff
@@ -48,6 +48,20 @@ defmodule GSearcher.SearchResults do
     |> Repo.update()
   end
 
+  def create_search_result_url(%SearchResult{id: id}, result_urls) when is_list(result_urls) do
+    Enum.each(result_urls, fn result_url ->
+      create_search_result_url(Map.merge(result_url, %{search_result_id: id}))
```

Comment:
Do we need to handle the error if one of result URL fails to create?

Reply:
I've put a log for this one for now.
I don't think it should output an `:error` as we don't want to roll back the entire job if one search result hasn't saved 🤔